Core/Graphics/Source/GraphicsImpl.h

```diff
@@ -21,7 +21,8 @@
 
 namespace Babylon
 {
-    struct GraphicsConfiguration;
+    struct WindowConfiguration;
+    struct ContextConfiguration;
```

Comment:
Is this sustainable? Is every platform going to have a comparable configuration struct for every construction scenario, or is it possible that some scenarios will be highly platform-specific such that we might end up with things like `AppleHeadlessNoContextConfiguration` for specialized scenarios? If the latter, that seems like an argument to keep the one-struct paradigm so that the platform specifics can be pushed out above and below this contract layer without exposing them here.